src/utils/reporting_workflow.js

```diff
@@ -45,13 +50,130 @@ const IG_REGISTRY = {
   [BASE_REPORTING_BUNDLE_PROFILE]: {} // base action definitions, see MEDMORPH-35
 };
 
-function initializeContext(planDefinition) {
-  // TODO -- see MEDMORPH-36
+/**
+ * Retrieve data from the EHR
+ *
+ * @param {string} uri - the search query for the data desired
+ * @returns axios promise with data
+ */
+async function readFromEHR(uri) {
+  const url = process.env.EHR;
```

Comment:
So originally I made it an env var because there would only be one EHR, but then Dylan pointed out it still needs to be in the servers collection anyway to store the authorization details. I decided against removing the env var but I'm thinking now there's no point to having it

Reply:
Once Mike's PR is merged I will add on to this storage/servers interface to implement a `getEHRServer()` method